Tests/test_file_png.py

```diff
@@ -786,6 +786,7 @@ def test_truncated_end_chunk(self):
         ImageFile.LOAD_TRUNCATED_IMAGES = True
         try:
             with Image.open("Tests/images/truncated_end_chunk.png") as im:
+                im.load()
```

Comment:
Why do you think it is necessary to add this?

Reply:
The test cases just now indicated that the new lines are not covered. 
My assumption is that **assert_image_equal_tofile** may not actually invoke load_end. However, I haven't looked into the code; I'm just testing whether the additions are covered.

Reply:
I just checked the implementation of assert_image_equal_tofile, and my guess seems to be incorrect. I need to rethink why the test cases are not covering it.

Reply:
The lack of coverage wasn't because `im.load()` wasn't there.

Because you are a first-time contributor to Pillow, our CI jobs mostly require manual approval by someone from the Pillow team before they are run. If the CI jobs are not run, this affects coverage. Approval had not been given yet.

Reply:
oh，I see. thanks for reponse.

Reply:
@radarhere  I have reset my wrong commit, i'am sorry for that.